src/main/java/Main.java

```diff
@@ -6,17 +6,17 @@ public class Main {
 
     public static void main(String[] args) throws IOException {
 
-        File file = new File("employees.csv");
+        File file = new File("src/main/employees.csv");
+        Factory factory = new Factory();
+        Employee[] employees = factory.employeesDataReader(file);
 
         if (file.exists()) {
-            File stats = new File("stats.txt");
-            stats.createNewFile();
+            File stats = new File("src/main/stats.txt");
 
-//            FileWriter fileWriter = new FileWriter(stats);
-//            fileWriter.write("Średnia wypłata: 5000");
-//
-//            fileWriter.close();
-        }
+            FileWriter fileWriter = new FileWriter(stats);
+            fileWriter.write(factory.printInfoAboutFactory(employees));
 
+            fileWriter.close();
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY